src/sheet-common.tsx

```diff
@@ -267,6 +270,19 @@ const Sheet = React.forwardRef<HTMLDivElement, SheetProps>(({onClose, children,
                                         bleedY
                                     />
                                 </div>
+                                {/**
+                                 * We put a button behind the top dragging area so that the sheet can
+                                 * be closed while navigating with the keyboard or with a screen reader.
+                                 */}
+                                <div className={styles.handleContainer}>
```

Comment:
Adding a focusable button "hidden" from the pointer devices (not accessible through clicking on it). This button is only accessible through keyboard/screen readers and when pressing it, it closes the sheet

Reply:
hmmm... if you are making this `display: none`, assistive technologies won't read it either. Have you tested this with a screen reader?

Reply:
I didn't use any `display: none` anywhere. That's why I say `"hidden" from pointer devices`. It's not really hidden, there is a layer on top of the button that prevents the click from reaching the button, nothing else